.golangci.yml

```diff
@@ -0,0 +1,5 @@
+issues:
+  exclude-rules:
+    - path: _test\.go
+      linters:
+        - errcheck
```

Comment:
should we still have it to ensure the tests aren't ignoring errors?

Reply:
Yeah, I can add it back in.